### ISSUE TITLE
Interrogation table error in .qmd with execute-dir: project

### ISSUE THREAD

**Issue by Aariq** [rstudio/pointblank#448]
## Prework

* [x] Read and agree to the [code of conduct](https://www.contributor-covenant.org/version/2/0/code_of_conduct/) and [contributing guidelines](https://github.com/rich-iannone/pointblank/blob/main/.github/CONTRIBUTING.md).
* [x] If there is [already a relevant issue](https://github.com/rich-iannone/pointblank/issues), whether open or closed, comment on the existing thread instead of posting a new issue.
* [x] Post a [minimal reproducible example](https://www.tidyverse.org/help/) so the maintainer can troubleshoot the problems you identify. A reproducible example is:
    * [x] **Runnable**: post enough R code and data so any onlooker can create the error on their own computer.
    * [x] **Minimal**: reduce runtime wherever possible and remove complicated details that are irrelevant to the issue at hand.
    * [x] **Readable**: format your code according to the [tidyverse style guide](https://style.tidyverse.org/).

## Description

I'm not 100% sure this is a pointblank bug and not a quarto bug, but starting here.

In a quarto project with `execute-dir: project`, interrogation tables in .qmd files not in the root directory cause the following error on render:

```
Quitting from lines 6-20 (test.qmd) 
Error in readLines(con, warn = FALSE) : cannot open the connection
Calls: .main ... <Anonymous> -> enumerate_output_formats -> read_utf8 -> readLines
```

If I do any of the following, the error goes away and the .qmd renders correctly:
- Move test.qmd to the project root directory
- Remove `execute-dir: project` from _quarto.yml
- use `quarto::quarto_render("dir/test.qmd", execute_dir = "dir")` to override the execute-dir YAML setting

## Reproducible example

Reprex here: https://github.com/Aariq/pointblank-debug. Open dir/test.qmd and click "render"

## Expected result

I wouldn't expect `pointblank` to care about the execute-dir. 

## Session info

<details> 
<summary>Session Info</summary>

```
R version 4.2.0 (2022-04-22)
Platform: x86_64-apple-darwin17.0 (64-bit)
Running under: macOS 13.0.1

Matrix products: default
LAPACK: /Library/Frameworks/R.framework/Versions/4.2/Resources/lib/libRlapack.dylib

locale:
[1] en_US.UTF-8/en_US.UTF-8/en_US.UTF-8/C/en_US.UTF-8/en_US.UTF-8

attached base packages:
[1] stats     graphics  grDevices utils     datasets  methods   base     

other attached packages:
[1] pointblank_0.11.2

loaded via a namespace (and not attached):
 [1] pillar_1.8.1       compiler_4.2.0     base64enc_0.1-3    tools_4.2.0        sys_3.4.1          digest_0.6.30     
 [7] gtable_0.3.1       jsonlite_1.8.3     lifecycle_1.0.3    tibble_3.1.8       pkgconfig_2.0.3    rlang_1.0.6       
[13] DBI_1.1.3          cli_3.4.1          rstudioapi_0.14    commonmark_1.8.1   curl_4.3.3         xfun_0.34         
[19] fastmap_1.1.0      withr_2.5.0        dplyr_1.0.10       httr_1.4.4         knitr_1.40         sass_0.4.2        
[25] generics_0.1.3     fs_1.5.2           vctrs_0.5.0        askpass_1.1        gitcreds_0.1.2     grid_4.2.0        
[31] rprojroot_2.0.3    gert_1.9.1         tidyselect_1.2.0   glue_1.6.2         here_1.0.1         R6_2.5.1          
[37] gh_1.3.1           fansi_1.0.3        blastula_0.3.2     ggplot2_3.4.0      purrr_0.3.5        magrittr_2.0.3    
[43] scales_1.2.1       credentials_1.3.2  htmltools_0.5.3    usethis_2.1.6.9000 gt_0.7.0           assertthat_0.2.1  
[49] colorspace_2.0-3   utf8_1.2.2         munsell_0.5.0      openssl_2.0.4      crayon_1.5.2    
```

</details>

**Comment by Aariq** [comment]
Related to #443 ?

**Comment by Aariq** [comment]
This was maybe fixed by updating to `gt` v0.8.0, but now I'm getting a different error: https://github.com/rich-iannone/pointblank/issues/450

**Comment by Aariq** [comment]
Ah ok, this appears to be fixed with the combination of `gt` v0.8.0 and the dev version of `pointblank`.  Sorry, didn't think of updating `gt` before!
